frontend/hub/collections/CollectionPage/CollectionContents.tsx

```diff
@@ -167,14 +168,10 @@ function RenderCommunityWarningMessage() {
   );
 }
 
-function returnPath() {
-  // TODO - this is maybe not correct, without documentation tab working, we cant be sure those paths are leading to right places
+function returnPath(item: CollectionContent) {
   // TODO - this is not handling insights well right now probably, but insights is still not discussed
-  let path = window.location.origin + window.location.pathname;
-  path = path.replace('/contents', '/documentation');
-
-  // TODO - this will work when documentation will be finished
-  //path += `/${content.content_type}/${content.name}/`;
+  let path = window.location.href;
+  path = path.replace('/contents', `/documentation/${item.content_type}/${item.name}`);
```

Comment:
We could leverage react-router-dom here to avoid to change the URL directly, I could not find many other places in the code basis where we manipulating the `window.location.href` directly like this one.

Reply:
I agree. Is there any method for this? I saw it in navigate, but not for static links.

Reply:
So I found getPageURL, I will use it :-)